test/unit/common_examples.rb

```diff
@@ -35,7 +39,7 @@
   context 'ssl is disabled' do
 
     before do
-      chef_run.node.set["sensu"]["use_ssl"] = false
+      chef_run.node.override["sensu"]["use_ssl"] = false
```

Comment:
node.normal is the equiv of node.set technically, but in the specs it doesn't really matter